### ISSUE TITLE
Input lose focus position after morphing

### ISSUE THREAD

**Issue by markuplab** [patrick-steele-idem/morphdom#21]
Please watch this video:
http://youtu.be/X33mMzkzR-A

```
HTML:
    <input type="text"/>

Javascript:
   var input = document.querySelector('input');
   input.addEventListener('input', function (e) {
       var newNode = document.createElement('input');
       newNode.value = e.currentTarget.value;
       morphdom(e.currentTarget, newNode);
  });
```

I know it's hard correct handle same cases, but it's very useful feature, because you can easy make two-way data-binding and don't care about "dirty hacks".

**Comment by patrick-steele-idem** [comment]
Thanks for reporting the problem. The simple fix was to not set the `value` property if it did not change. It looks like Chrome (and maybe other browsers) would change the caret position even if the new value was the same as the old.

New version published: morphdom@1.0.2

**Comment by markuplab** [comment]
Thank you! Now it's work correct.
